tests/test_check_strings.py

```diff
@@ -0,0 +1,125 @@
+import unittest
```

Comment:
Can we name this file `test_check_po_files.py` instead?

Reply:
renamed